gneiss/regression/_mixedlm.py

```diff
@@ -221,3 +220,132 @@ def percent_explained(self):
         # instead of population variance (df=0).
         axis_vars = np.var(self.balances, ddof=1, axis=0)
         return axis_vars / axis_vars.sum()
+
+    def coefficients(self, tree=None):
+        """ Returns coefficients from fit.
+        Parameters
+        ----------
+        tree : skbio.TreeNode, optional
+            The tree used to perform the ilr transformation.  If this
+            is specified, then the prediction will be represented as
+            proportions. Otherwise, if this is not specified, the prediction
+            will be represented as balances. (default: None).
+
+        Returns
+        -------
+        pd.DataFrame
+            A table of coefficients where rows are covariates,
+            and the columns are balances. If `tree` is specified, then
+            the columns are proportions.
+        """
+        coef = pd.DataFrame()
+
+        for r in self.results:
+            c = r.params
+            c.name = r.model.endog_names
+            coef = coef.append(c)
+
+        if tree is not None:
+            basis, _ = balance_basis(tree)
+            c = ilr_inv(coef.values.T, basis=basis).T
+
+            return pd.DataFrame(c, index=[n.name for n in tree.tips()],
+                                columns=coef.columns)
+        else:
+            return coef.T
```

Comment:
This nasty code is a result of this #187
If we can code up the multivariate linear mixed effects, we don't need statsmodels and can seriously clean up / speed up this code.